includes/story-export/readme.txt

```diff
@@ -0,0 +1,9 @@
+=== AMP Stories Export Archive ===
```

Comment:
I would suggest to rephrase to something like that:

> _This folder contains the HTML as well as all the assets for a story exported by the WordPress AMP Stories editor (https://wordpress.org/plugins/amp/). The base URL is set according to the AMP plugin's story base URL setting._
> 
> _You can use this folder to host the generated story on another server. Note that using HTTPS is recommended._

@MackenzieHartung could you perhaps proof read the above?

Reply:
Sounds good to me.

We should also create a documentation page on amp-wp.org and point to that URL in this readme.

There we can add screenshots, instructions, etc.

Reply:
The revisions @ThierryA made look good to me. I am assuming, however, that we will be moving the bulk of this content to the amp-wp.org site, and point to there from the readme as @swissspidy mentioned. Correct?

Reply:
Yes. I think we'll keep like the first sentence or so, and then add the link to the docs.